definitions/pipelines/rnaseq.cwl

```diff
@@ -155,3 +163,10 @@ steps:
             bam: index_bam/indexed_bam
         out:
             [metrics, chart]
+    cgpBigWig_bamcoverage:
```

Comment:
It'd be my personal preference if we keep step names to snake_case rather than capitalizing some letters.

Reply:
Ive made this change.